### ISSUE TITLE
code has Intel architecture specific element

### ISSUE THREAD

**Issue by wwortel** [ISDN4Linux/mISDN#5]
In mISDN/drivers/isdn/mISDN/octvqe/octvqe_liniux.c a timestamp is used that uses the call in line 534
pChan->ulTimestampIn = (unsigned long)rdtsc();

rdtsc however is the reading of an Intel architecture specific time register that does not exist in other CPU architectures.
Tried to compile this for an ARM based platform and only could get there by removing these calls (am not using octvqe anyway).

**Comment by kkeil** [comment]
Yes the code was developed for Intel only years ago and I'm not so familar with this specific stuff (echo canceling). If somebody comes up with a CPU neutral Version I'm glade to merge it.
